lib/go/test/flow_epoch_test.go

```diff
@@ -148,9 +148,44 @@ func TestEpochPhaseMetadataChange(t *testing.T) {
 		rewardIncreaseFactor)
 
 	t.Run("Should be able to change the configurable metadata during the staking auction", func(t *testing.T) {
+		// Should fail to set epoch config with invalid config sum of the staking phase and dkg phases is greater than epoch
+		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateUpdateEpochConfigScript(env), idTableAddress)
+		_ = tx.AddArgument(cadence.NewUInt64(5)) // dkg
+		_ = tx.AddArgument(cadence.NewUInt64(5)) // staking
+		_ = tx.AddArgument(cadence.NewUInt64(5)) // epoch
+		signAndSubmit(
+			t, b, tx,
+			[]flow.Address{b.ServiceKey().Address, idTableAddress},
+			[]crypto.Signer{b.ServiceKey().Signer(), IDTableSigner},
+			true,
+		)
+
+		// Should set epoch config successfully when increasing the epochs views
+		tx = createTxWithTemplateAndAuthorizer(b, templates.GenerateUpdateEpochConfigScript(env), idTableAddress)
+		_ = tx.AddArgument(cadence.NewUInt64(2)) // staking
+		_ = tx.AddArgument(cadence.NewUInt64(4)) // dkg
```

Comment:
I think these comments are backward now
```suggestion
		_ = tx.AddArgument(cadence.NewUInt64(2)) // dkg
		_ = tx.AddArgument(cadence.NewUInt64(4)) // staking
```

Reply:
https://github.com/onflow/flow-core-contracts/pull/282/commits/f8d069626de3b7ea5c6ea7f7d408f8b804b33d06